semgrep-core/perf/run-perf-suite

```diff
@@ -87,6 +89,10 @@ def run_tests(
 
         pattern_file = os.path.join(dirname, test.name + ext)
         target_file = os.path.join(dirname, test.target)
+        target_file_gz = os.path.join(dirname, test.target + ".gz")
+
+        if os.path.isfile(target_file_gz):
+            subprocess.check_call(["gunzip", "-f", target_file_gz])
```

Comment:
so the uncompressed file will be in perf/input/? Should we ignore it in .gitignore then?

Reply:
We'll have to ignore by by one, but I will do.